gatsby-node.js

```diff
@@ -16,43 +16,83 @@ exports.onCreateNode = ({ node, actions, getNode }) => {
 exports.createPages = ({ graphql, actions }) => {
   const { createPage } = actions;
 
-  // const talkTemplate = path.resolve(`./src/templates/Talk/index.jsx`);
+  const talkTemplate = path.resolve(`./src/templates/Talk/index.jsx`);
   const storyTemplate = path.resolve(`./src/templates/Story/index.jsx`);
   // const speakerTemplate = path.resolve(`./src/templates/Speaker/index.jsx`);
   // const meetupTemplate = path.resolve(`./src/templates/Meetup/index.jsx`);
 
-  return graphql(
-    `
-      {
-        stories: allMarkdownRemark(
-          filter: { fileAbsolutePath: { regex: "/stories/" } }
-        ) {
-          nodes {
-            fields {
-              slug
+  return Promise.all([
+    graphql(
+      `
+        {
+          stories: allMarkdownRemark(
+            filter: { fileAbsolutePath: { regex: "/stories/" } }
+          ) {
+            nodes {
+              fields {
+                slug
+              }
             }
           }
         }
+      `
+    ).then(result => {
+      if (result.errors) {
+        throw result.errors;
       }
-    `
-  ).then(result => {
-    if (result.errors) {
-      throw result.errors;
-    }
-
-    // Create blog posts pages.
-    const {
-      stories: { nodes: stories },
-    } = result.data;
-
-    stories.forEach(story => {
-      createPage({
-        path: `/story${story.fields.slug}`,
-        component: storyTemplate,
-        context: {
-          slug: story.fields.slug,
+
+      // Create blog posts pages.
+      const {
+        stories: { nodes: stories },
+      } = result.data;
+
+      stories.forEach(story => {
+        createPage({
+          path: `/story${story.fields.slug}`,
+          component: storyTemplate,
+          context: {
+            slug: story.fields.slug,
+          },
+        });
+      });
+    }),
+    graphql(
+      `
+        {
+          talks: github {
+            repository(owner: "react-knowledgeable", name: "talks") {
+              issues(first: 100) {
```

Comment:
If we are just getting issues with the `talk` label, we can further narrow this:

```suggestion
              issues(first: 100, labels: ["talk"]) {
```

Reply:
Yes, you right. Let's me update it.